lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -273,6 +273,26 @@ function sendUpdateValueNgsi2(entityName, originMeasures, originTypeInformation,
             return initObj;
         }
     }
+    //it returns a flat object from an Metadata Object
```

Comment:
Is this comment still valid? What "flat" refers here?

Maybe better this one?

```suggestion
    //it returns a metadata object using the same structure described
    // at https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#metadata-support
```